libs/hdf-converters/src/base-converter.ts

```diff
@@ -10,6 +10,7 @@ export interface ILookupPath {
   path?: string | string[];
   transformer?: (value: any) => unknown;
   arrayTransformer?: (value: unknown[], file: any) => unknown[];
+  pathTransform?: (value: unknown, file: any) => unknown;
```

Comment:
these changes will probably need to be rebased on top of baseconverter depending on which pr gets merged first

Reply:
rebased code